server/src/test/java/org/apache/cassandra/sidecar/config/SidecarConfigurationTest.java

```diff
@@ -112,7 +112,7 @@ void testReadingJmxConfiguration() throws IOException
         SidecarConfiguration config = SidecarConfigurationImpl.readYamlConfiguration(yamlPath);
         assertThat(config.serviceConfiguration().jmxConfiguration()).isNotNull();
         JmxConfiguration jmxConfiguration = config.serviceConfiguration().jmxConfiguration();
-        assertThat(jmxConfiguration.maxRetries()).isEqualTo(42);
+        assertThat(jmxConfiguration.maxRetries()).isEqualTo(1);
```

Comment:
We all like the ultimate answer, but since it has been modified in the ymls, update the test assertion accordingly.